as yet unclassified
handleError: anException withResult: aResult

	testMaster handleSlaveError: anException fromSlave: self